pymesync.py

```diff
@@ -788,6 +807,24 @@ def __create_or_update(self, parameters, identifier,
             # Request error
             return [{self.error: e}]
 
+    def __interpret_times(self, duration):
+        """When a time_entry is created, a user will enter a time duration as
+           one of the parameters of the object. This method will convert that
+           entry (if it's entered as a string) into the appropriate integer
+           equivalent (in seconds).
+        """
+        try:
+            t = time.strptime(duration, "%Hh%Mm")
+            hours_spent = t.tm_hour
+            minutes_spent = t.tm_min
+
+            # Convert duration to seconds
+            seconds = (hours_spent * 3600) + (minutes_spent * 60)
+            return seconds
+        except:
+            t = None
```

Comment:
Consider setting the error message here, instead of the method that calls it (i.e. ``create_time`` and ``update_time``). That way if we need to expand/change our error messages we only need to do it in one place. 

Then you return whatever error message you generate to the calling method, which passes that on to the user.

Another benefit is that if we ever add to this method and need more error messages, we can put the logic for that here.